jupyterlab/commands.py

```diff
@@ -1762,8 +1762,12 @@ def _yarn_config(logger):
     -------
     {"yarn config": dict, "npm config": dict} if unsuccessfull the subdictionary are empty
     """
-    node = which('node')
     configuration = {"yarn config": {}, "npm config": {}}
+    try:
+        node = which('node')
+    except ValueError:  # Node not found == user with no need for building jupyterlab
+        return configuration
```

Comment:
Is there ever any time it would be useful to see this error? Should we `logger.debug` it?

Reply:
I was hesitating to print it. The error message would be:

```
ValueError: Please install nodejs 5+ and npm before continuing installation. nodejs may be installed using conda or directly from the nodejs website.
```
This could mislead somebody asking the debug trace but with no need for node to be installed. What about printing a message like:
```python
logger.debug("NodeJS was not found. Yarn user configuration is ignored.")
```